### DISCUSSION TITLE
Display withdrawable amount in NFT SVG

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/evm-monorepo#725] [General]
From [BlockEnthusiast](https://twitter.com/BlockEnthusiast/status/1722364045213225255):

> Would be great if it would visually represent the current balance held and the already streamed balance
> 
> Else hard to really determine bid price without going to a block explorer and trying to sort out balances from the contract

**Comment by andreivladbrg** [comment]
> current balance held and the already streamed balance

I'll assume that _current balance held = refundable amount_ and _already streamed balance = withdrawable amount_.

We are already displaying the "streamed balance":
<img width="125" alt="image" src="https://github.com/sablier-labs/v2-core/assets/99738872/a90962f2-4dd3-4b38-bc2d-6d7c348b7aa0">

Meaning that he is asking for the refundable amount, right?

**Reply by PaulRBerg** [reply]
> current balance held = refundable amount and already streamed balance = withdrawable amount.

Yes

> We are already displaying the "streamed balance":

Not anymore since https://github.com/sablier-labs/v2-core/pull/692

> he is asking for the refundable amount, right?

As you said, he is asking for the refundable amount and the withdrawable amount.

---

I'm skeptical that it is worth modifying the SVG to contain such granular data because of the image caching issue on OpenSea. The solution is to have specialized trading UIs.

**Reply by andreivladbrg** [reply]
> Not anymore since https://github.com/sablier-labs/v2-core/pull/692

That is about to change when we deploy `2.1`, currently it still has the "Streamed" card.

> I'm skeptical that it is worth modifying the SVG to contain such granular data because of the image caching issue on OpenSea. 

Agree.

> The solution is to have specialized trading UIs.

Yes, but it is a long way until we build something like this 😅

**Reply by PaulRBerg** [reply]
> it is a long way until we build something like this

Correct. But we have to always open discussions about feature requests made by users.

**Comment by PaulRBerg** [comment]
Some more feedback from another (potential) user: [Hana](https://twitter.com/hanamizuki) from Perpetual Protocol.

The cancelability of a stream should be visible in the NFT.

**Comment by PaulRBerg** [comment]
Closing due to competing priorities. NFTs haven't been a popular feature in Sablier.
